lib/rules/value-keyword-case/__tests__/index.js

```diff
@@ -1938,3 +1938,91 @@ testRule(rule, {
     }
   ]
 });
+
+testRule(rule, {
+  ruleName,
+  config: ["lower", { ignoreProperties: ["display", "/^(b|B)ackground$/"] }],
+
+  accept: [
+    {
+      code: "a { display: bloCk; }"
+    },
+    {
+      code: "a { display: BloCk; }"
+    },
+    {
+      code: "a { display: BLOCK; }"
+    },
+    {
+      code: "a { display: block; }"
+    },
+    {
+      code: "div { background: Red; }"
```

Comment:
Use `a` rather than `div` for consistency please, otherwise a future maintainer might think the different selector is significant to the test when it is not.